spec/serializers/user_serializer_spec.rb

```diff
@@ -69,23 +66,15 @@
       end
     end
 
-    context "with filled out card background" do
-      before do
-        user.user_profile.card_background = 'http://card.com'
-      end
-
-      it "has a profile background" do
-        expect(json[:card_background]).to eq 'http://card.com'
-      end
-    end
-
-    context "with filled out profile background" do
+    context "with filled out backgrounds" do
       before do
-        user.user_profile.profile_background = 'http://background.com'
+        user.user_profile.upload_card_background(upload)
+        user.user_profile.upload_profile_background(upload)
```

Comment:
It might be worth using two different uploads in order to make sure that the URLs aren't mixed up.